docs/userdocs/k8s/deploy-submarine.md

```diff
@@ -0,0 +1,70 @@
+
+
+# Deploy Submarine On K8s
+
+## Deploy Submarine using Helm Chart (Recommended)
+
+### Create images
+submarine server
+```bash
+./dev-support/docker-images/submarine/build.sh
+```
+
+submarine database
+```bash
+./dev-support/docker-images/database/build.sh
+```
+
+### install helm
+For more info see https://helm.sh/docs/intro/install/
+
+### Deploy Submarine Server, mysql
+You can modify some settings in ./helm-charts/submarine/values.yaml
+```bash
+helm install submarine ./helm-charts/submarine
+```
+
+### Delete deployment
+```bash
+helm delete submarine
+```
+
+FIXME: TF Operator / PyTorch Operator can be deployed by Helm Chart, correct?
+
+## Deploy Submarine Manually
+
+### Get package
+You can dowload submarine from releases page or build from source.
+
+### Configuration
+`KUBE_CONFIG` environment need to be specified when deploy Submarine server.
+
+### Start Submarine Server
+Running the submarine server, executing the following command:
+```
+# if build from source. You need to run this under the target dir like submarine-dist/target/submarine-dist-0.4.0-SNAPSHOT-hadoop-2.9/submarine-dist-0.4.0-SNAPSHOT-hadoop-2.9/
+./bin/submarine-daemon.sh start getMysqlJar
+```
+
+The REST API URL is: `http://127.0.0.1:8080/api/v1/jobs`
+
+### Deploy Tensorflow Operator
+For more info see [deploy tensorflow operator](./ml-frameworks/tensorflow.md).
+
+### Deploy PyTorch Operator
+```bash
+cd <submarine_code_path_root>/dev-support/k8s/pytorchjob
+./deploy-pytorch-operator.sh
+
+```
+
+## Post Deploy
+
+### Setup port-forward {host port}:{container port}
+
+FIXME: Why we need this? Need elaborate
```

Comment:
I think it only used in manual deployment to export the submarine service outside the K8s cluster, and we should support to automatically operate in helm charts.

Reply:
@jiwq @wangdatan  IIUC, the user's local port forwarding is needed even if deploy submarine in k8s.

Reply:
Agree, but we had two ways to deploy the submarine in K8s, helm chart and manually. I mean the submarine chart should support deploy the port-forward feature.

Reply:
Thought more, it may make more sense to not write the manual deploy Submarine option.
If the Helm chart is available, user should always use Helm chart. What is the benefit of leaving manual deployment option here? 

I just updated my local doc to remove the manual deploy option, I can revert it back if it doesn't make sense. 

Port forwarding seems like an additional step. I suggest to remove it.